src/bindings/dune

```diff
@@ -0,0 +1,27 @@
+(library
+ (name interop)
+ (modules interop)
+ (preprocess
+  (pps gen_js_api.ppx))
+ (libraries core_kernel js_of_ocaml gen_js_api)
+ (modes byte))
+
+(library
+ (name node)
+ (modules node)
+ (preprocess
+  (pps gen_js_api.ppx))
+ (js_of_ocaml
+  (javascript_files node_stub.js))
+ (libraries interop core_kernel promise_jsoo jsonoo gen_js_api)
+ (modes byte))
+
+(library
+ (name vscode)
+ (modules vscode vscode__core vscode__languageclient)
```

Comment:
Why are you manually using the `__` convention here? this is just a convention chosen by dune, we shouldn't use it.